README.md

```diff
@@ -132,8 +132,12 @@ assertRecyclerViewItemCount(R.id.recycler, 10);
 // And another tricky feature
 assertThatBackButtonClosesTheApp();
 
-// Is this ImageView showing this drawable?
+// Is this ImageView showing a drawable?
+assertDrawable(R.id.image_view);
 assertDrawable(R.id.image_view, R.drawable.ic_barista);
+
+// ...or not?
+assertWithoutDrawable(R.id.image_view);
```

Comment:
@Sloy we should move this entry to the ones that have the style "blablabla / ...or not?". It's the moment to do it? Can we do it later? (My vote for later)

Reply:
Later is good for me